docs/pages/Introduction/bridges.mdx

```diff
@@ -0,0 +1,69 @@
+- Fostering collaboration among developers from diverse blockchain ecosystems to create innovative products.
+- Drawing users and communities from various ecosystems to their dApps. For instance, a DeFi protocol deployed on Ethereum could attract users from Binance Smart Chain by offering cross-chain asset transfers through a bridge.
+
+{ /* write five benefits of bridges */}
+
+
+
+## Types of bridges
+At a high level, bridges can be categorized as trusted and trustless.
+
+{   /* Broad category */}
+
+- **Oracle based bridges(Trusted)**: These are infrastructure solutions that enable interoperability and communication between disparate blockchain networks by leveraging external data sources known as oracles. These bridges facilitate the exchange of information and assets between blockchains that may have different protocols and consensus mechanisms. They rely on trusted set of accounts or entities to authorise and verify transactions and state transitions across interconnected networks.
+
+
+- **Consensus based bridges(Trustless)**: These are infrastructure solutions that enables interoperability between different blockchain networks through a mechanism based on achieving consensus among the participating networks. Unlike oracle-based bridges, which rely on external data sources to facilitate communication between blockchains, consensus-based bridges use consensus algorithms to ensure the validity and integrity of transactions and state transitions across interconnected networks. This is the most secure way to connect two networks.
+
+
+## Factors to consider when choosing a bridge
+Bridges are susceptible to hacks, they are part of the top three biggest hacks in DEFI.
+
+- **Speed**: In the domain of bridges, there's a fundamental trade-off between security and speed. Oracle-based bridges, which rely on trusted entities to verify transactions and state transitions across interconnected networks, boast faster transaction speeds. However, this expedited processing comes at the expense of security, leaving them susceptible to attacks and manipulation, such as the release of private keys. One key takeaway is, If the speed of message passing surpasses the consensus speed of interacting networks, the security isn't guaranteed by the networks' finality itself.
+
+- **Security**: On the contrary, consensus-based bridges prioritize security over speed, leveraging consensus mechanisms amongst participating networks to validate transactions and ensure integrity. For instance, Hyperbridge operates on this model, where the time taken for asset bridging is contingent on the duration of consensus in the participating networks. This approach eliminates the need for trusted entities and instead relies on consensus proofs and state transition validity to communicate with other networks, thus preventing hacks and attacks. 
+
```

Comment:
Add ability to self relay and also decentralized relayers which improves censorship resistance